caltechdata_api/customize_schema.py

```diff
@@ -508,11 +508,11 @@ def validate_metadata(json_record):
                 if creator["nameType"] == "Organizational":
                     if "name" not in creator:
                         errors.append("Each organizational 'creator' must have 'name'.")
-            else:
-                if "familyName" not in creator:
-                    errors.append(
-                        "Each 'creator' must have a 'familyName' or have type Organizational"
-                    )
+                else:
```

Comment:
No, this shouldn't be changed. Users should either provide a nameType or familyName (in which case we assume type Personal)